app/src/test/java/ch/hikemate/app/model/profile/ProfileRepositoryFirestoreTest.kt

```diff
@@ -66,7 +105,6 @@ class ProfileRepositoryFirestoreTest {
     `when`(mockDocumentSnapshot.getTimestamp("joinedDate")).thenReturn(testTimestamp)
 
     val profile = repository.documentToProfile(mockDocumentSnapshot)
-    assert(profile != null)
```

Comment:
Question: Is there a reason to remove this ? It would crash on the next line anyway.